streamlit_cropper/__init__.py

```diff
@@ -151,6 +154,11 @@ def st_cropper(img: Image, realtime_update: bool=True, box_color: str='blue', as
         cropped_img = img.crop((rect['left'], rect['top'], rect['width'] + rect['left'], rect['height'] + rect['top']))
         return cropped_img
     elif return_type.lower() == 'box':
+        # Return scaled box according to resize ratio
+        rect['left'] = int(rect['left'] * resized_ratio_w)
+        rect['width'] = int(rect['width'] * resized_ratio_w)
+        rect['top'] = int(rect['top'] * resized_ratio_h)
+        rect['height'] = int(rect['height'] * resized_ratio_h)
```

Comment:
Moving the rescaling of the box to before the `if` (line 151) would also solve the issue noted in https://github.com/turner-anderson/streamlit-cropper/pull/9#issuecomment-1001946105 &ndash; I'm guessing that most people don't expect cropping to also downsample the image. This would also require changing line 154 to work on the full-size image.